docs/tutorials/using_apis/Mbed_USB_WAV_Audio_Player/Mbed_USB_WAV_Audio_Player.md

```diff
@@ -1,12 +1,16 @@
 # Mbed USB WAV audio player
 
-This tutorial explains how to put together a USB WAV audio player with Mbed. You will load a WAV file onto an SD card connected to an Mbed board. You can then play the WAV file from the board on a host computer over USB. Once connected, the Mbed board acts as a USB microphone. The sound that the microphone is "recording" is actually sound data read from the SD card and sent to the host computer. This allows the sound to be played on speakers or headphones or recorded with a program, such as Audacity.
+<!--since the library isn't called Mbed USB, just USB, and since this lives in the Mbed docs - can we remove "mbed" from the title?-->
+
+This tutorial explains how to put together a USB WAV audio player with Mbed OS. You will load a WAV file onto an SD card connected to an Mbed board, and then connect that board to your computer. The Mbed board acts as the computer's USB microphone. The sound that the microphone is "recording" is the data on the SD card that it sends to the host computer. This allows the sound to be played on speakers or headphones or recorded with a program, such as Audacity.
+
+<!--this is more an example than a tutorial; I'm not really learning a whole lot here. If there as a discussion about some of the choices made in the code, that would be interesting.-->
```

Comment:
Possible 2nd part to the intro:

The example code below uses the following libraries in addition to mbed.h:
1. USBAudio.h for accessing functions associated with USB audio.
1. SDBlockDevice.h to make use of the SD card.
1. FATFileSystem.h to set up a FAT file system on the SD card.
1. AudioPlayer.h to access functions associated with reading and writing audio data.
1. WaveAudioStream.h to access functions associated with WAV files.

We begin by creating a SPI connection to the SD card (sd object) and by creating a FAT filesystem on the SD card (fs object). We set the SPI frequency to 25000000 Hz. Next, we open our song (WAV) file that we have previously stored onto the SD card with the file.open() function. The string input parameter to this function must match the name of the file as it is on the SD card ("songs/Bach-minuet-in-g.wav" in our case). Next, we check that our file is a valid WAV file by making it into a WaveAudioStream object and checking with the song.get_valid() function. Next, we check that the WAV file is 16-bit by making sure that the song.get_bytes_per_sample() returns 2 (2 bytes). Next, we create our USBAudio object that allows us to write the data from the WAV file to the computer. We initialize this object by giving it the parameters 'true' in order to connect on initialization, a receive frequency of 8000 Hz, the number of channels utilized by the WAV file, a transmit frequency that matches the sample rate of the WAV file, and the number of channels utilized by the WAV file. We then proceed to write all of the data from the WAV file to the computer over USB. This is done by reading data from the WAV file into a 1500-byte buffer, and then writing the contents of that buffer to the computer via the audio.write() function until we reach the end of the data in the WAV file. Last, we close our WAV file object, and the program exits.